semver.md

```diff
@@ -327,7 +330,7 @@ The Semantic Versioning specification is authored by [Tom
 Preston-Werner](http://tom.preston-werner.com), inventor of Gravatars and
 cofounder of GitHub.
 
-If you'd like to leave feedback, please [open an issue on
+If you'd like to leave feedback, please [open an Issue on
```

Comment:
I was considering “Issues” a proper noun (as in “GitHub Issues”).  Will change back to lowercase.